plugin.video.nhlgcl/changelog.txt

```diff
@@ -0,0 +1,186 @@
+2017.2.24
```

Comment:
changelog is no longer parsed by kodi in favour of the news tag. Maybe, since your changelog is outdated, consider to drop it?